Dockerfile

```diff
@@ -3,7 +3,7 @@ FROM registry.access.redhat.com/ubi9/ubi-minimal as builder
 
 RUN microdnf install -y make tar gzip which && microdnf clean all
 
-RUN curl -L https://go.dev/dl/go1.20.11.linux-amd64.tar.gz | tar -C /usr/local -xzf -
```

Comment:
@akrejcir @0xFelix How do you suggest to check if it's arm64 or amd64 here? Isn't architecture usually based on base images?

Reply:
There may be some way to detect which architecture is used and download the right package based on that.
This was the reason why I dropped my original [PR](https://github.com/kubevirt/ssp-operator/pull/485). I didn't want to spend too much time figuring this out, if no one would use it.